R/zchunk_L125.bcoc_unmgd_R_S_T_Y.R

```diff
@@ -63,14 +67,16 @@ module_emissions_L125.bcoc_unmgd_R_S_T_Y <- function(command, ...) {
 
     # Define function to map from EDGAR to GCAM, and select year 2000,
     # rename the value column by forest fire type, and add a Non.CO2 emission species
-    edgar_to_gcam <- function(x, fire_type, em_name){ x %>%
-      mutate(iso = tolower(Country), Country = NULL) %>%
+    edgar_to_gcam <- function(x, fire_type, em_name) {
+      Country <- GCAM_region_ID <- `2000` <- NULL # silence package check notes
```

Comment:
This shouldn't be necessary.  Mentioning all of these in the list on lines 39-42 (which `Country` isn't, but the others are) should do the trick because of R's lexical scoping.  My preference would be to have all the note-silencing to happen in one go at the top of the function so that someone reading the code can more easily ignore it.